frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/StartHandler.java

```diff
@@ -152,6 +153,13 @@ public APIGatewayProxyResponseEvent handleRequest(
                     "reauthenticateHeader: {} reauthenticate: {}",
                     reauthenticateHeader,
                     reauthenticate);
+
+            var oldSessionId =
+                    getHeaderValueFromHeaders(
+                            input.getHeaders(),
+                            OLD_SESSION_ID_HEADER,
+                            configurationService.getHeadersCaseInsensitive());
+            LOG.info("oldSessionId: {}", oldSessionId);
```

Comment:
I wonder if we'll need to make this a POST request and use the body? I think we're going to send a lot like this, and headers feels like an abuse